packages/python/readme_metrics/tests/PayloadBuilder_test.py

```diff
@@ -54,26 +77,147 @@ def compareRequests(self, fromReq, fromPayload):
         # Compare the two contents and check if they are similar(?)
         return True
 
-    @pytest.mark.skip(reason="@todo")
-    def testBlackListed(self, url):
+    def mockMiddlewareConfig(self, **kwargs):
+        return MetricsApiConfig(
+            "koSyKkViOR5gD6yjBxlsprHfjAIlWOh6",
```

Comment:
```suggestion
            "apiKey",
```

Reply:
haha sure, this was just copy pasta from preexisting test